Source/Scene/FrameState.js

```diff
@@ -277,6 +277,13 @@ function FrameState(context, creditDisplay, jobScheduler) {
    */
   this.terrainExaggeration = 1.0;
 
+  /**
+   * The height relative to which terrain is exaggerated.
+   * @type {Number}
+   * @default 0.0
+   */
+  this.terrainExaggerationRelativeHeight = 0.0;
```

Comment:
`FrameState` still needs exaggeration values because `GlobeSurfaceTileProvider` doesn't have access to the `Globe`, but does have access to `FrameState`. This is used when checking if the exaggeration has changed between frames so that geodetic surface normals can be added or removed.